packages/l10n/scripts/generate-l10n-data.js

```diff
@@ -294,6 +296,114 @@ const mapDiffToWarnings = (oldData, newData) => {
     .filter(Boolean);
 };
 
+const updateTimeZoneData = async (key) => {
+  const allTimeZoneIds = moment.tz.names();
+  // uncomment for testing/review purposes, will be removed once PR is out of draft
+  // const fakeTimeZones = ['zzspace/moon', 'zzspace/mars', 'zzspace/jupiter'];
+  // const allTimeZoneIds = moment.tz.names().concat(fakeTimeZones);
+  const dataFolderPath = path.join(__dirname, '..', DATA_DIR[key].path);
+  const sourceDataFilePath = path.join(dataFolderPath, 'core.json');
+  await fs.accessSync(sourceDataFilePath, fs.constants.F_OK);
+  const translatedTimeZones = JSON.parse(
+    fs.readFileSync(sourceDataFilePath, { encoding: 'utf8' })
+  );
+  const translatedTimeZoneIds = Object.keys(translatedTimeZones);
+  console.log(
+    `[${key}]: Checking moment-timezone for new IANA timezone identifiers`
+  );
+  const newTimeZoneIds = allTimeZoneIds.filter(
+    (id) =>
+      !(translatedTimeZoneIds.includes(id) || EXCLUDED_TIME_ZONES.includes(id))
+  );
+  /** If there are IANA timezone identifiers returned by moment-timezone
+   *  that are not in the translations or exclusions list,
+   *  we must get user input as to whether those time zones should be:
+   *  a) translated,
+   *  b) excluded from being displayed/translated, or
+   *  c) ignored until the next time this script is run.
+   */
+  if (newTimeZoneIds.length) {
+    console.log(
+      `\n[${key}]: Found ${chalk.cyan(
+        newTimeZoneIds.length
+      )} new time zones: ${newTimeZoneIds.map(
+        (id) => ` ${chalk.underline(chalk.cyan(id))}`
+      )}\n`
+    );
+    // parse user feedback for each timezone
+    const results = await parseNewTimeZones(newTimeZoneIds);
+
+    if (results.addToTranslations.length) {
+      // add timezones to be tranlated to translations object
+      const updatedTranslationData = results.addToTranslations.reduce(
+        (translationData, newTranslationId) => {
+          return {
+            ...translationData,
+            [newTranslationId]: 'ADD NEW TRANSLATION',
+          };
+        },
+        translatedTimeZones
+      );
+      // sort translations alphabetically
+      const updatedTranslationsFile = Object.keys(updatedTranslationData)
+        .sort((a, b) => a.toLowerCase().localeCompare(b.toLowerCase()))
+        .reduce((acc, key) => {
+          return {
+            ...acc,
+            [key]: updatedTranslationData[key],
+          };
+        }, {});
+      // write new translations file
+      shell.mkdir('-p', dataFolderPath);
```

Comment:
I don't think this is needed to use this package. We can just `fs.mkdirSync(dir);`

Reply:
`fs.mkdirSync(dir)` errors out if the directory already exists, `shell.mkdir('-p', dir)` does not

Reply:
Sure, I can check for existence using `fs`.

I'm following this pattern because it's what's used in `updateLocaleData`, would you like me to remove `shell` there as well so that we can remove the dependency altogether?

Reply:
That's for another time I think. A decent goal though to avoid such bulky things for so little benefit.